proto/internalapi/scanner/v4/matcher_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+option go_package = "v4";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+import "internalapi/scanner/v4/vulnerability_report.proto";
+
+package v4;
+
+message GetVulnerabilitiesRequest {
+    string hash_id = 1;
+}
+
+message Metadata {
+    google.protobuf.Timestamp LastVulnerabilityUpdate = 5;
+}
+
+// Matcher finds vulnerabilities in index reports.
+service Matcher {
+
+    // GetVulnerabilities get or create a VulnerabilityReport and return it.
```

Comment:
nit: `GetVulnerabilities gets a VulnerabilityReport....`

I don't think users need to know the details (also I think a new one would be generated each time, as I don't think they are stored)